plugins/sublime/sublime.plugin.zsh

```diff
@@ -41,6 +41,30 @@ elif  [[ "$OSTYPE" = darwin* ]]; then
             break
         fi
     done
+
+elif  [[ "$OSTYPE" == cygwin ]]; then
+    local cygprogfiles=""
+    cygprogfiles=$(cygpath "$PROGRAMFILES")
+    local cygprogfiles_x86=""
+    cygprogfiles_x86=$(cmd /c "echo %ProgramFiles(x86)%")
+    cygprogfiles_x86=${cygprogfiles_x86/[\\r\\n]//}
+    cygprogfiles_x86=$(cygpath "$cygprogfiles_x86")
+    local _sublime_cygwin_paths > /dev/null 2>&1
+    _sublime_cygwin_paths=(
+        "$cygprogfiles/Sublime Text 2/sublime_text.exe"
+        "$cygprogfiles/Sublime Text 3/sublime_text.exe"
+        "$cygprogfiles_x86/Sublime Text 2/sublime_text.exe"
+        "$cygprogfiles_x86/Sublime Text 3/sublime_text.exe"
+    )
+
+    IFS=$'\n'
```

Comment:
This change to `$IFS` really ought to be `local`, since it could affect other input processing if it leaks.